andela/mrm_api
# chore(code-climate): integrate code-climate with backend

#### What does this PR do?
>- Allow for Code Climate comments and reporting

#### How should this be manually tested?
>- Open an existing pull request or create a new one.
>- Code climate automatically run the tests and check for test coverage on the PR as shown on image 2.
>- The badges are updated on the readme on develop branch as shown on image 1

#### Relevant Pivotal Tracker Stories
[158227258](<https://www.pivotaltracker.com/n/projects/2154921/stories/158227258>)

#### Screenshots?
Badges [image 1]
![screen shot 2018-06-21 at 5 13 20 pm](https://user-images.githubusercontent.com/22421283/41731995-c447d10a-7577-11e8-83fe-51b046755b83.png)
<img width="473" alt="screen shot 2018-06-21 at 12 38 59 pm" src="https://user-images.githubusercontent.com/22421283/41732002-cce6f8f4-7577-11e8-90c2-09ca7c9c1a32.png">
On pull request [image 2]
<img width="719" alt="screen shot 2018-06-21 at 12 39 55 pm" src="https://user-images.githubusercontent.com/22421283/41732005-cd9c4452-7577-11e8-8717-67781ab9f30d.png">
